hail/python/hailtop/utils/utils.py

```diff
@@ -678,9 +690,16 @@ def is_transient_error(e):
     if isinstance(e, asyncio.TimeoutError):
         return True
     if (isinstance(e, aiohttp.ClientConnectorError)
-            and hasattr(e, 'os_error')
             and is_transient_error(e.os_error)):
         return True
+    if (isinstance(e, aiohttp.ClientOSError)
+            and len(e.args) >= 2
+            and e.args[0] == 1
```

Comment:
I thought the change that added this already went in. Must have been in a different location?

Reply:
You are correct, good catch. I screwed up a rebase. Removed. Originally added here: https://github.com/hail-is/hail/pull/13817/files